__tests__/SelectionBox.test.tsx

```diff
@@ -23,7 +23,7 @@ describe("Selection box tests", () => {
 		fireEvent.press(getByLabelText("Show selections"));
 
 		// Assert
-		expect(getAllByText(placeTypeLabels[0])).toHaveLength(1);
+		expect(getAllByText(enabledFiltersMap[0].label)).toHaveLength(1);
```

Comment:
this may need a `//eslint-ignore-next-line`, I'm seeing that `@typescript-eslint/no-unsafe-member-access` is acting up